lte/gateway/c/core/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -1006,13 +1006,15 @@ status_code_e sgw_handle_modify_bearer_request(
               "for bearer %d\n",
               eps_bearer_ctxt_p->s_gw_teid_S1u_S12_S4_up,
               eps_bearer_ctxt_p->eps_bearer_id);
-          // This is best effort, ignore return code.
-          gtp_tunnel_ops->send_end_marker(enb, modify_bearer_pP->teid);
-          // delete GTPv1-U tunnel
-          rv = gtp_tunnel_ops->del_tunnel(
-              enb, enb_ipv6, ue_ipv4, ue_ipv6,
-              eps_bearer_ctxt_p->s_gw_teid_S1u_S12_S4_up,
-              eps_bearer_ctxt_p->enb_teid_S1u, NULL);
+          if (gtp_tunnel_ops) {
```

Comment:
what is this change needed for? is `gtp_tunnel_ops` not valid on the unit tests?

Reply:
Yes, in one of these tests, gtp_tunnel_ops is not initialized. Therefore, I added this check to make sure the safety. I am asking @ssanadhya for her feedback. Let's see if we have a better way.

Reply:
@thhung , I have used the `#if !MME_UNIT_TEST` directive to skip calls to gtp_tunnel_ops for unit testing. Please use the same here.

I have a work in progress PR https://github.com/magma/magma/pull/10702 to create a mock GTP tunnel for unit testing. In the mean time let's use `#if !MME_UNIT_TEST`.